src/main/java/blusunrize/immersiveengineering/common/util/compat/crafttweaker/managers/MineralMixManager.java

```diff
@@ -66,12 +67,13 @@ public RecipeType<MineralMix> getRecipeType()
 	 * @docParam background <block:minecraft:white_wool>
 	 */
 	@ZenCodeType.Method
-	public void addRecipe(String recipePath, List<Percentaged<IItemStack>> outputs, int weight, float failChance, ResourceLocation[] dimensions, Block background)
+	public void addRecipe(String recipePath, List<Percentaged<IItemStack>> outputs, List<Percentaged<IItemStack>> spoils, int weight, float failChance, ResourceLocation[] dimensions, Block background)
 	{
 		final ResourceLocation resourceLocation = new ResourceLocation("crafttweaker", recipePath);
 		final StackWithChance[] stacksWithChances = outputs.stream().map(CrTIngredientUtil::getStackWithChance).toArray(StackWithChance[]::new);
+		final StackWithChance[] spoilsWithChances = spoils.stream().map(CrTIngredientUtil::getStackWithChance).toArray(StackWithChance[]::new);
 		final List<ResourceKey<Level>> dimensionKeys = Arrays.stream(dimensions).map(resourceLocation1 -> ResourceKey.create(Registry.DIMENSION_REGISTRY, resourceLocation1)).collect(Collectors.toList());
-		final MineralMix mix = new MineralMix(resourceLocation, stacksWithChances, weight, failChance, dimensionKeys, background);
+		final MineralMix mix = new MineralMix(resourceLocation, stacksWithChances, spoilsWithChances, weight, failChance, dimensionKeys, background);
```

Comment:
Has Craftweaker integration for this been tested?

Reply:
Yes! It worked well.